### PR TITLE
Edited Chapter 9

### PR DESCRIPTION
**Pull request by amandamuledy**
Erik has some suggestions in his commit, too.

Line 128--I made this more aggressive change because your sentence starting "Another aspect of this decision" is going to make it hard for readers to connect the dots.

171--It isn't clear what "that link" is. I think you should specify.

381--I always hate to fill in blanks like this. But I know you often describe things as interesting, so I thought I would take that route in my guess. Please correct if you have a more specific term (or if I'm wrong).

965--Your link to Through the Looking Glass doesn't work. May I suggest https://sfu-db.github.io/dbsystems/Lectures/OLTP_Through_the_Looking_Glass.pdf?

General Feeback:
Again, "we" is confusing in this chapter since you use "we" interchangeably as "we, you and I (author and reader) who are together following these instructions" and "we, the folks at RavenDB, who have programmed the way query results are returned" 

LastPRoduct" is in here a few times. I don't want to mess with your snippets, but did you mean "product"?

### CODE REVIEW

**Comment by ayende** [comment]
Line 128 - reads much better

File: Ch09/Ch09.md [hunk 1]
```diff
@@ -472,42 +441,33 @@ where Boost(Address.City = 'London', 3) or
...
+document modeling (cohesive, coherent and isolated) is the notion that a document is cohesive. From a modeling perspective, it doesn't usually make sense 
 to just grab some pieces of data from a document without having it all there.
 
-At least, it doesn't make sense until you realize that you often need to _display_ the data to the user, you often need to pick and choose what will be 
-shown. For example, while an order without its order lines isn't really meaningful in a business sense, to the user, knowing that an order was made on 
-Dec 17th is probably enough to recall that they had to get expediated shipping on their last minute holidays shopping to get it in time. 
+At least, it doesn't make sense until you realize that you often need to _display_ the data to the user, picking and choosing what will be shown. For example, while an order without its order lines isn't really meaningful in a business sense to the user, knowing that an order was made on Dec 17th is probably enough to recall they required expedited shipping on their last-minute holiday shopping to get it in time.
```
Comment (ayende) [inline Ch09/Ch09.md:467]: The second sentence seems very long and awkward.
"while an order isn't really meaningful to the user" ? aren't we missing a comma or some connecting word here?

**Comment by ayende** [comment]
* Regarding line comments, it is great.
* `LastPRoduct` Is actually `LastProduct` (as in the last of the products)
* My update commit is here: https://github.com/ravendb/book/commit/7546d1a2cdf0caf3cb8710b39c71ee75ec2af956
This also includes a new section on spatial queries that need editing as well.

**Comment by amandamuledy** [comment]
Hmm. I think it reads okay. Since I consider "to the user" is essential to
the meaning of the phrase it's tied to (which I'm assuming is "meaningful
in a business sense"), I wouldn't set it off with punctuation. I'm
following rules for restrictive/nonrestrictive clauses.

Either way, I'm never one to argue with breaking up long sentences. It's
one of my favorite pastimes.

How about...

At least, it doesn't make sense until you realize that you often need to
_display_ the data to the user, picking and choosing what will be shown.
For example, an order without its order lines may not be very meaningful in
a business sense to the user. But knowing that an order was made on Dec
17th is probably enough information to recall they required expedited
shipping on their last-minute holiday shopping to get it in time.


I might have not totally understood what you meant, but this should help
you know where to break it up in a way that makes sense. I'll check out
what you do if you send me a link to your revision.

**Comment by amandamuledy** [comment]
Okay! I inserted comments where you need changes. Let me know if you have
anything else you need me to look at over the next few days.

--Amanda

**Comment by ayende** [comment]
Thanks, here are the fixes: https://github.com/ravendb/book/commit/a4eb02b2bbfbfe62007a7f3a7ad750c2655b8ab0

**Pull request merged**